gno.land/pkg/gnoland/genesis.go

```diff
@@ -245,3 +245,19 @@ func DefaultGenState() GnoGenesisState {
 	}
 	return gs
 }
+
+func ValidateGenState(state GnoGenesisState) error {
```

Comment:
Can you add a test for this (invalid module gen state)? 🙏

Reply:
Done: https://github.com/gnolang/gno/pull/4031/commits/a8c1761062c46d16422969b61064ff7be1a98904